assets/js/modules/idea-hub/components/dashboard/DashboardIdeasWidget/Idea.js

```diff
@@ -253,6 +329,16 @@ Idea.propTypes = {
 	buttons: PropTypes.arrayOf( PropTypes.string ).isRequired,
 };
 
+Idea.notices = {
```

Comment:
I like the idea of using a map for this but let's not attach it to the component as a static property unless it's needed outside of the component (in which case it could also maybe just be an import).